.github/workflows/build.yaml

```diff
@@ -43,9 +43,9 @@ jobs:
           echo ${{ secrets.GCLOUD_SERVICE_KEY }} | base64 --decode --ignore-garbage > "$HOME"/gcloud.json
           GOOGLE_APPLICATION_CREDENTIALS="$HOME"/gcloud.json web-platform/functions/node_modules/.bin/firebase deploy --only functions,firestore --project ${{secrets.PROJECT_ID_STAGE}} -c web-platform/firebase.json
 
-      - name: Archive Attention Stream zip
+      - name: Archive Mozilla Rally extension zip
         uses: actions/upload-artifact@v3
         with:
-          name: attention-stream
+          name: rally-extension
           path: |
-              extensions/attention-stream/web-ext-artifacts/
\ No newline at end of file
+              extensions/rally-extension/web-ext-artifacts/
```

Comment:
This doesn't seem to be working, did you plan on changing the dir name here but decided not to?

Reply:
haha yep; I actually went through the whole process of changing the the dir name and all references and then ran into the Glean probe scraper problem so I changed it all back; this is a holdover - thanks for catching that!